src/frontends/tensorflow_common/src/op/matrix_band_part.cpp

```diff
@@ -0,0 +1,71 @@
+#include "openvino/op/logical_and.hpp"
+#include "openvino/op/less_equal.hpp"
+#include "openvino/op/unsqueeze.hpp"
+#include "openvino/op/constant.hpp"
+#include "openvino/op/concat.hpp"
+#include "openvino/op/reshape.hpp"
+#include "utils.hpp"
+
+using namespace std;
+using namespace ov::op;
+
+namespace ov {
+namespace frontend {
+namespace tensorflow {
+namespace op {
+
```

Comment:
please add declaration of `translate_matrix_band_part_op` in `openvino\src\frontends\tensorflow_common\include\common_op_table.hpp`